docs/CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ For more information about `AlexsLemonade/scpca-nf` versions, please see [the re
 <!-- PUT THE NEW CHANGELOG ENTRY RIGHT BELOW THIS -->
 <!-------------------------------------------------->
 
+## PLACEHOLDER FOR METADATA ADDITION IN REPORTS
+
+* A table containing sample metadata (e.g., age, sex, diagnosis) is now available in both the QC report (`qc.html`) and the supplemental cell type report (`celltype-report.html`) included in all downloads.
```

Comment:
I added `etc.` to be a little clear that there are more than those three. Is it too much with eg and etc?

```suggestion
* A table containing sample metadata (e.g., age, sex, diagnosis, etc.) is now available in both the QC report (`qc.html`) and the supplemental cell type report (`celltype-report.html`) included in all downloads.
```

Reply:
I don't think you need `etc.` if you are listing examples.